encoding/jsonbq/marshal.go

```diff
@@ -26,7 +26,19 @@ type Field struct {
 // The JSON conforms to the format in this example:
 // https://cloud.google.com/bigquery/docs/nested-repeated#example_schema
 func MarshalSchema(schema bigquery.Schema) ([]byte, error) {
+	return MarshalOptions{}.MarshalSchema(schema)
+}
+
+type MarshalOptions struct {
+	// Set to true to marshal json with indentation for better human readability
+	Indent bool
```

Comment:
`Indent` should be a `string` to be consistent with the `protojson` API:

https://pkg.go.dev/google.golang.org/protobuf/encoding/protojson#MarshalOptions

Reply:
Nice! Will change that.